roles/keycloak/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
       - version
   register: mysql_ready
   until: mysql_ready is not failed
-  retries: 60
```

Comment:
can you make this a separate change please?

Reply:
https://github.com/vexxhost/atmosphere/pull/1409